components/prism-pug.js

```diff
@@ -112,6 +112,14 @@
 			pattern: /(^[\t ]*)(?!-)[\w\-#.]*[\w\-](?:(?:&[^(]+)?\([^)]+\))*\/?:?/m,
 			lookbehind: true,
 			inside: {
+				'attr-id': {
+					pattern: /(^[\t ]*(?!-)[\w\-.]*)#[\w\-]+/,
```

Comment:
The lookbehind seems really complicated. From my understanding, [ids and classes](https://pugjs.org/language/attributes.html) aren't matches right now, right? So couldn't we just match them _after_ everything else? Wouldn't inserting the following after `'punctuation': /:/` also work:

```js
'attr-id': /#[\w\-]+/,
'attr-class': /\.[\w\-]+/
```

Or am I missing something?